Run java example throw event trace data source exception

When running java example, it will throw event trace data source exception.

The main class is `org.apache.shardingsphere.elasticjob.lite.example.JavaMain`

Exception is:

```
[ERROR] 2020-07-05 19:15:40,200 --javaDataflowElasticJob_Worker-1-- [org.apache.shardingsphere.elasticjob.lite.tracing.JobEventBus] Elastic job: create tracing listener failure, error is:  
org.apache.shardingsphere.elasticjob.lite.tracing.exception.TracingConfigurationException: org.h2.jdbc.JdbcSQLException: Table "JOB_EXECUTION_LOG" already exists; SQL statement:
CREATE TABLE JOB_EXECUTION_LOG (id CHARACTER(40) NOT NULL, job_name CHARACTER(100) NOT NULL, task_id CHARACTER(255) NOT NULL, hostname CHARACTER(255) NOT NULL, ip CHARACTER(50) NOT NULL, sharding_item INTEGER NOT NULL, execution_source CHARACTER(20) NOT NULL, failure_cause CHARACTER VARYING(4000) NULL, is_success INTEGER NOT NULL, start_time TIMESTAMP NULL, complete_time TIMESTAMP NULL, PRIMARY KEY (id)) [42101-184]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.listener.RDBTracingListenerConfiguration.createTracingListener(RDBTracingListenerConfiguration.java:37) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.listener.RDBTracingListenerConfiguration.createTracingListener(RDBTracingListenerConfiguration.java:30) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.listener.TracingListenerFactory.getListener(TracingListenerFactory.java:56) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.JobEventBus.register(JobEventBus.java:68) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.JobEventBus.<init>(JobEventBus.java:56) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.<init>(LiteJobFacade.java:71) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:82) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:69) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.createExecutor(LiteJob.java:56) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:52) [classes/:na]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202) [quartz-2.3.2.jar:na]
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:na]
Caused by: org.h2.jdbc.JdbcSQLException: Table "JOB_EXECUTION_LOG" already exists; SQL statement:
CREATE TABLE JOB_EXECUTION_LOG (id CHARACTER(40) NOT NULL, job_name CHARACTER(100) NOT NULL, task_id CHARACTER(255) NOT NULL, hostname CHARACTER(255) NOT NULL, ip CHARACTER(50) NOT NULL, sharding_item INTEGER NOT NULL, execution_source CHARACTER(20) NOT NULL, failure_cause CHARACTER VARYING(4000) NULL, is_success INTEGER NOT NULL, start_time TIMESTAMP NULL, complete_time TIMESTAMP NULL, PRIMARY KEY (id)) [42101-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.message.DbException.get(DbException.java:179) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.message.DbException.get(DbException.java:155) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:111) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.command.CommandContainer.update(CommandContainer.java:78) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.command.Command.executeUpdate(Command.java:254) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:198) ~[h2-1.4.184.jar:1.4.184]
	at org.apache.commons.dbcp.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:172) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:172) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.storage.RDBJobEventStorage.createJobExecutionTable(RDBJobEventStorage.java:132) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.storage.RDBJobEventStorage.createJobExecutionTableAndIndexIfNeeded(RDBJobEventStorage.java:100) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.storage.RDBJobEventStorage.initTablesAndIndexes(RDBJobEventStorage.java:91) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.storage.RDBJobEventStorage.<init>(RDBJobEventStorage.java:74) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.listener.RDBTracingListener.<init>(RDBTracingListener.java:36) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.listener.RDBTracingListenerConfiguration.createTracingListener(RDBTracingListenerConfiguration.java:35) ~[classes/:na]
	... 11 common frames omitted
[ERROR] 2020-07-05 19:15:40,200 --javaSimpleJob_Worker-1-- [org.apache.shardingsphere.elasticjob.lite.tracing.JobEventBus] Elastic job: create tracing listener failure, error is:  
org.apache.shardingsphere.elasticjob.lite.tracing.exception.TracingConfigurationException: org.h2.jdbc.JdbcSQLException: Table "JOB_EXECUTION_LOG" already exists; SQL statement:
CREATE TABLE JOB_EXECUTION_LOG (id CHARACTER(40) NOT NULL, job_name CHARACTER(100) NOT NULL, task_id CHARACTER(255) NOT NULL, hostname CHARACTER(255) NOT NULL, ip CHARACTER(50) NOT NULL, sharding_item INTEGER NOT NULL, execution_source CHARACTER(20) NOT NULL, failure_cause CHARACTER VARYING(4000) NULL, is_success INTEGER NOT NULL, start_time TIMESTAMP NULL, complete_time TIMESTAMP NULL, PRIMARY KEY (id)) [42101-184]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.listener.RDBTracingListenerConfiguration.createTracingListener(RDBTracingListenerConfiguration.java:37) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.listener.RDBTracingListenerConfiguration.createTracingListener(RDBTracingListenerConfiguration.java:30) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.listener.TracingListenerFactory.getListener(TracingListenerFactory.java:56) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.JobEventBus.register(JobEventBus.java:68) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.JobEventBus.<init>(JobEventBus.java:56) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.<init>(LiteJobFacade.java:71) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:82) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:69) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.createExecutor(LiteJob.java:56) [classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:52) [classes/:na]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202) [quartz-2.3.2.jar:na]
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:na]
Caused by: org.h2.jdbc.JdbcSQLException: Table "JOB_EXECUTION_LOG" already exists; SQL statement:
CREATE TABLE JOB_EXECUTION_LOG (id CHARACTER(40) NOT NULL, job_name CHARACTER(100) NOT NULL, task_id CHARACTER(255) NOT NULL, hostname CHARACTER(255) NOT NULL, ip CHARACTER(50) NOT NULL, sharding_item INTEGER NOT NULL, execution_source CHARACTER(20) NOT NULL, failure_cause CHARACTER VARYING(4000) NULL, is_success INTEGER NOT NULL, start_time TIMESTAMP NULL, complete_time TIMESTAMP NULL, PRIMARY KEY (id)) [42101-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.message.DbException.get(DbException.java:179) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.message.DbException.get(DbException.java:155) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:111) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.command.CommandContainer.update(CommandContainer.java:78) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.command.Command.executeUpdate(Command.java:254) ~[h2-1.4.184.jar:1.4.184]
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:198) ~[h2-1.4.184.jar:1.4.184]
	at org.apache.commons.dbcp.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:172) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:172) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.storage.RDBJobEventStorage.createJobExecutionTable(RDBJobEventStorage.java:132) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.storage.RDBJobEventStorage.createJobExecutionTableAndIndexIfNeeded(RDBJobEventStorage.java:100) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.storage.RDBJobEventStorage.initTablesAndIndexes(RDBJobEventStorage.java:91) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.storage.RDBJobEventStorage.<init>(RDBJobEventStorage.java:74) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.listener.RDBTracingListener.<init>(RDBTracingListener.java:36) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.tracing.rdb.listener.RDBTracingListenerConfiguration.createTracingListener(RDBTracingListenerConfiguration.java:35) ~[classes/:na]
	... 11 common frames omitted
```

--- a/elastic-job-lite-tracing/elastic-job-lite-tracing-rdb/src/main/java/org/apache/shardingsphere/elasticjob/lite/tracing/rdb/storage/RDBStorageSQLMapper.java
+++ b/elastic-job-lite-tracing/elastic-job-lite-tracing-rdb/src/main/java/org/apache/shardingsphere/elasticjob/lite/tracing/rdb/storage/RDBStorageSQLMapper.java
@@ -74,7 +74,7 @@ public final class RDBStorageSQLMapper {
     }
     
     private InputStream getPropertiesInputStream(final String sqlPropertiesFileName) {
-        InputStream sqlPropertiesFile = RDBJobEventStorage.class.getClassLoader().getResourceAsStream(String.format("META-INF/sql/%s.properties", sqlPropertiesFileName));
+        InputStream sqlPropertiesFile = RDBJobEventStorage.class.getClassLoader().getResourceAsStream(String.format("META-INF/sql/%s", sqlPropertiesFileName));
         return null == sqlPropertiesFile ? RDBJobEventStorage.class.getClassLoader().getResourceAsStream("META-INF/sql/sql92.properties") : sqlPropertiesFile;
     }
 }

--- a/elastic-job-lite-tracing/elastic-job-lite-tracing-rdb/src/main/resources/META-INF/sql/h2.properties
+++ b/elastic-job-lite-tracing/elastic-job-lite-tracing-rdb/src/main/resources/META-INF/sql/h2.properties
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-JOB_EXECUTION_LOG.TABLE.CREATE=CREATE TABLE JOB_EXECUTION_LOG (id VARCHAR(40) NOT NULL, job_name VARCHAR(100) NOT NULL, task_id VARCHAR(255) NOT NULL, hostname VARCHAR(255) NOT NULL, ip VARCHAR(50) NOT NULL, sharding_item INT NOT NULL, execution_source VARCHAR(20) NOT NULL, failure_cause VARCHAR(4000) NULL, is_success INT NOT NULL, start_time TIMESTAMP NULL, complete_time TIMESTAMP NULL, PRIMARY KEY (id))
+JOB_EXECUTION_LOG.TABLE.CREATE=CREATE TABLE IF NOT EXISTS JOB_EXECUTION_LOG  (id VARCHAR(40) NOT NULL, job_name VARCHAR(100) NOT NULL, task_id VARCHAR(255) NOT NULL, hostname VARCHAR(255) NOT NULL, ip VARCHAR(50) NOT NULL, sharding_item INT NOT NULL, execution_source VARCHAR(20) NOT NULL, failure_cause VARCHAR(4000) NULL, is_success INT NOT NULL, start_time TIMESTAMP NULL, complete_time TIMESTAMP NULL, PRIMARY KEY (id))
 
 JOB_EXECUTION_LOG.INSERT=INSERT INTO JOB_EXECUTION_LOG (id, job_name, task_id, hostname, ip, sharding_item, execution_source, is_success, start_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)
 JOB_EXECUTION_LOG.INSERT_COMPLETE=INSERT INTO JOB_EXECUTION_LOG (id, job_name, task_id, hostname, ip, sharding_item, execution_source, is_success, start_time, complete_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
@@ -23,8 +23,8 @@ JOB_EXECUTION_LOG.INSERT_FAILURE=INSERT INTO JOB_EXECUTION_LOG (id, job_name, ta
 JOB_EXECUTION_LOG.UPDATE=UPDATE JOB_EXECUTION_LOG SET is_success = ?, complete_time = ? WHERE id = ?
 JOB_EXECUTION_LOG.UPDATE_FAILURE=UPDATE JOB_EXECUTION_LOG SET is_success = ?, complete_time = ?, failure_cause = ? WHERE id = ?
 
-JOB_STATUS_TRACE_LOG.TABLE.CREATE=CREATE TABLE JOB_STATUS_TRACE_LOG (id VARCHAR(40) NOT NULL, job_name VARCHAR(100) NOT NULL, original_task_id VARCHAR(255) NOT NULL, task_id VARCHAR(255) NOT NULL, slave_id VARCHAR(50) NOT NULL, source VARCHAR(50) NOT NULL, execution_type VARCHAR(20) NOT NULL, sharding_item VARCHAR(100) NOT NULL, state VARCHAR(20) NOT NULL, message VARCHAR(4000) NULL, creation_time TIMESTAMP NULL, PRIMARY KEY (id))
-TASK_ID_STATE_INDEX.INDEX.CREATE=CREATE INDEX TASK_ID_STATE_INDEX ON JOB_STATUS_TRACE_LOG (task_id, state)
+JOB_STATUS_TRACE_LOG.TABLE.CREATE=CREATE TABLE IF NOT EXISTS JOB_STATUS_TRACE_LOG (id VARCHAR(40) NOT NULL, job_name VARCHAR(100) NOT NULL, original_task_id VARCHAR(255) NOT NULL, task_id VARCHAR(255) NOT NULL, slave_id VARCHAR(50) NOT NULL, source VARCHAR(50) NOT NULL, execution_type VARCHAR(20) NOT NULL, sharding_item VARCHAR(100) NOT NULL, state VARCHAR(20) NOT NULL, message VARCHAR(4000) NULL, creation_time TIMESTAMP NULL, PRIMARY KEY (id))
+TASK_ID_STATE_INDEX.INDEX.CREATE=CREATE INDEX IF NOT EXISTS TASK_ID_STATE_INDEX ON JOB_STATUS_TRACE_LOG (task_id, state)
 
 JOB_STATUS_TRACE_LOG.INSERT=INSERT INTO JOB_STATUS_TRACE_LOG (id, job_name, original_task_id, task_id, slave_id, source, execution_type, sharding_item, state, message, creation_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
 JOB_STATUS_TRACE_LOG.SELECT=SELECT * FROM JOB_STATUS_TRACE_LOG WHERE task_id = ?

--- a/elastic-job-lite-tracing/elastic-job-lite-tracing-rdb/src/main/resources/META-INF/sql/mysql.properties
+++ b/elastic-job-lite-tracing/elastic-job-lite-tracing-rdb/src/main/resources/META-INF/sql/mysql.properties
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-JOB_EXECUTION_LOG.TABLE.CREATE=CREATE TABLE JOB_EXECUTION_LOG (id VARCHAR(40) NOT NULL, job_name VARCHAR(100) NOT NULL, task_id VARCHAR(255) NOT NULL, hostname VARCHAR(255) NOT NULL, ip VARCHAR(50) NOT NULL, sharding_item INT NOT NULL, execution_source VARCHAR(20) NOT NULL, failure_cause VARCHAR(4000) NULL, is_success INT NOT NULL, start_time TIMESTAMP NULL, complete_time TIMESTAMP NULL, PRIMARY KEY (id))
+JOB_EXECUTION_LOG.TABLE.CREATE=CREATE TABLE IF NOT EXISTS JOB_EXECUTION_LOG (id VARCHAR(40) NOT NULL, job_name VARCHAR(100) NOT NULL, task_id VARCHAR(255) NOT NULL, hostname VARCHAR(255) NOT NULL, ip VARCHAR(50) NOT NULL, sharding_item INT NOT NULL, execution_source VARCHAR(20) NOT NULL, failure_cause VARCHAR(4000) NULL, is_success INT NOT NULL, start_time TIMESTAMP NULL, complete_time TIMESTAMP NULL, PRIMARY KEY (id))
 
 JOB_EXECUTION_LOG.INSERT=INSERT INTO JOB_EXECUTION_LOG (id, job_name, task_id, hostname, ip, sharding_item, execution_source, is_success, start_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)
 JOB_EXECUTION_LOG.INSERT_COMPLETE=INSERT INTO JOB_EXECUTION_LOG (id, job_name, task_id, hostname, ip, sharding_item, execution_source, is_success, start_time, complete_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
@@ -23,7 +23,7 @@ JOB_EXECUTION_LOG.INSERT_FAILURE=INSERT INTO JOB_EXECUTION_LOG (id, job_name, ta
 JOB_EXECUTION_LOG.UPDATE=UPDATE JOB_EXECUTION_LOG SET is_success = ?, complete_time = ? WHERE id = ?
 JOB_EXECUTION_LOG.UPDATE_FAILURE=UPDATE JOB_EXECUTION_LOG SET is_success = ?, complete_time = ?, failure_cause = ? WHERE id = ?
 
-JOB_STATUS_TRACE_LOG.TABLE.CREATE=CREATE TABLE JOB_STATUS_TRACE_LOG (id VARCHAR(40) NOT NULL, job_name VARCHAR(100) NOT NULL, original_task_id VARCHAR(255) NOT NULL, task_id VARCHAR(255) NOT NULL, slave_id VARCHAR(50) NOT NULL, source VARCHAR(50) NOT NULL, execution_type VARCHAR(20) NOT NULL, sharding_item VARCHAR(100) NOT NULL, state VARCHAR(20) NOT NULL, message VARCHAR(4000) NULL, creation_time TIMESTAMP NULL, PRIMARY KEY (id))
+JOB_STATUS_TRACE_LOG.TABLE.CREATE=CREATE TABLE IF NOT EXISTS JOB_STATUS_TRACE_LOG (id VARCHAR(40) NOT NULL, job_name VARCHAR(100) NOT NULL, original_task_id VARCHAR(255) NOT NULL, task_id VARCHAR(255) NOT NULL, slave_id VARCHAR(50) NOT NULL, source VARCHAR(50) NOT NULL, execution_type VARCHAR(20) NOT NULL, sharding_item VARCHAR(100) NOT NULL, state VARCHAR(20) NOT NULL, message VARCHAR(4000) NULL, creation_time TIMESTAMP NULL, PRIMARY KEY (id))
 TASK_ID_STATE_INDEX.INDEX.CREATE=CREATE INDEX TASK_ID_STATE_INDEX ON JOB_STATUS_TRACE_LOG (task_id(128), state)
 
 JOB_STATUS_TRACE_LOG.INSERT=INSERT INTO JOB_STATUS_TRACE_LOG (id, job_name, original_task_id, task_id, slave_id, source, execution_type, sharding_item, state, message, creation_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)

--- a/pom.xml
+++ b/pom.xml
@@ -37,6 +37,7 @@
         <module>elastic-job-lite-spring</module>
         <module>elastic-job-lite-lifecycle</module>
         <module>elastic-job-lite-console</module>
+        <module>examples</module>
     </modules>
     
     <properties>

